### PR TITLE
Add rate limiting scaling parameter

### PR DESCRIPTION
**Pull request by bowei**
Allows global scaling of all rate limits via command line flags

### CODE REVIEW

File: pkg/ratelimit/ratelimit.go [hunk 1]
```diff
@@ -154,7 +156,15 @@ func constructRateLimitImpl(params []string) (flowcontrol.RateLimiter, error) {
 		if err != nil {
 			return nil, fmt.Errorf("invalid argument for rate limiter type %v. Expected %v to be a int.", rlType, implArgs[1])
 		}
-		return flowcontrol.NewTokenBucketRateLimiter(float32(qps), burst), nil
+		oldQPS := qps
+		oldBurst := burst
+		if flags.F.GCERateLimitScale != 1.0 {
```
Comment (freehan) [inline pkg/ratelimit/ratelimit.go:161]: do you want to validate if `flags.F.GCERateLimitScale > 1` ? 

or make `GCERateLimitScale`'s type to be `uint`?

Comment (bowei) [reply pkg/ratelimit/ratelimit.go:161]: we can floor it to 1.0

**Review by freehan — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**